src/effects.js

```diff
@@ -27,19 +27,29 @@ var
 	rfxtypes = /^(?:toggle|show|hide)$/,
 	rrun = /queueHooks$/;
 
-function raf() {
+function raf( timestamp ) {
 	if ( timerId ) {
 		window.requestAnimationFrame( raf );
-		jQuery.fx.tick();
+		jQuery.fx.tick( timestamp );
 	}
 }
 
+// We need to be using jQuery.now() or performance.now() consistently as they return different
+// values: performance.now() counter starts on page load.
+// Support: IE <10, Safari <8.0, iOS <9, Android <4.4, Node with jsdom 9.4
+function getTimestamp() {
```

Comment:
I could put the `if` outside of the function definition but that only adds 3 bytes and I suppose such a single `if` will not cause any grief for any relatively modern browser.

The fact that we can't reliably mock stuff *before* jQuery is loaded is obviously a problem but this is an issue bigger than this PR.